cloud-sql/sqlserver/mssql/index.js

```diff
@@ -50,32 +51,8 @@ async function accessSecretVersion(secretName) {
   return version.payload.data;
 }
 
-// [START cloud_sql_sqlserver_mssql_create]
```

Comment:
Snippet bot is failing because we're removing this sample and region tags.

Are you sure we're not breaking any links by removing?